tests/multitest.py

```diff
@@ -26,6 +26,14 @@ def eqtest(a1: (np.ndarray, int, float), a2,
     return np.allclose(a1, a2, rtol=rtol, atol=atol)
 
 
+@overload  # noqa: F811
+def eqtest(x: type(None), y, **kwargs):
+    if y is None:
+        return True
+    else:
+        return y.sum().item() == 0
```

Comment:
I think we want to check that `y` is all zeros, not that its sum is zero. The code for that would be `(y == 0).all()`.